src/renderer/src/App.tsx

```
@@ -218,10 +218,16 @@
     }
   }, [detectedFileFormat, outFormatLocked, setFileFormat, setOutFormatLocked]);
 
+  const previousPlaybackVolume = useRef(playbackVolume);
   const toggleMuted = useCallback(() => {
-    if (playbackVolume === 0) setPlaybackVolume(1);
-    if (playbackVolume === 1) setPlaybackVolume(0);
-  }, [playbackVolume, setPlaybackVolume]);
+    setPlaybackVolume(volume => {
```

ESLint

[error] Expected parentheses around arrow function argument.

(arrow-parens)